util/objectHelpers.js

```diff
@@ -0,0 +1,8 @@
+/*
+ * Helper functions for dealing with objects.
+ */
+
+exports.get = require('lodash.get')
+exports.set = require('lodash.set')
+exports.union = require('lodash.union')
+exports.keys = require('lodash.keys')
```

Comment:
Since this entire repo is essentially for utils, how about making this sit in something like `helpers/objectHelper.js`